scripts/set-default-contact/index.js

```diff
@@ -10,37 +10,39 @@ const compileUrl = path => {
   }
 };
 
+const fetchUrl = async (url, options) => {
+  options = {
+    ...options,
+    headers: {
+      'Authorization': 'Basic ' + btoa(`${url.username}:${url.password}`),
+      'Content-Type': 'application/json'
+    }
+  };
+  url.username = '';
+  url.password = '';
+
+  const response = await fetch(url.toString(), options);
+  if (response.ok) {
+    return await response.json();
+  }
+
+  response.message = await response.text();
+  throw response;
+};
+
 const getChtUsers = async () => {
-  console.log('   Using COUCH_URL env var:', process.env.COUCH_URL);
-  console.log('');
   //  pin  to v1 of API so it is backwards compatible with CHT 3.x
   const url = compileUrl('/api/v1/users');
-  const options = {
-    uri: url.href,
-    json: true
-  };
-
-  return request.get(options)
-    .then(users => {
-      if (typeof users === 'object' ){
-        console.log('   Found', users.length, 'users\n');
-        return users;
-      }
-    });
+  const users = await fetchUrl(url);
+  if (typeof users === 'object') {
+    console.log(`   Found ${users.length} users\n`);
+    return users;
+  }
 };
 
 const getObjectFromMedicDb = async id => {
   const url = compileUrl('/medic/' + id);
-  const options = {
-    uri: url.href,
-    json: true
-  };
-
-  return request.get(options)
-    .then(object => {
-      return object;
-    });
-
+  return fetchUrl(url);
```

Comment:
```suggestion
  return await fetchUrl(url);
```
or 
remove async from the function declaration